environment.yml

```diff
@@ -1,54 +1,97 @@
+# environment.yml - conda environment specification file for RMG-Py
+#
+# Usage: conda env create --file environment.yml
+#
+# This file contains all of the software packages needed to run RMG-Py.
+# There is a mixture of the following
+# - packlages would could be installed at OS level, but we install here
+#   for better version control
+# - python tools
+# - external software tools specific to chemistry
+# - other software we maintain which RMG depends on
+# + some other categories (see below)
+#
+# Changelog:
+# - May 15, 2023 Added this changelog, added inline documentation,
+#   made depdency list more explicit (@JacksonBurns).
+#
 name: rmg_env
 channels:
   - defaults
   - rmg
   - conda-forge
   - cantera
 dependencies:
+# System-level dependencies - we could install these at the OS level
+# but by installing them in the conda environment we get better control
   - cairo
   - cairocffi
+  - ffmpeg
+  - xlrd
+  - xlwt
+  - h5py
+  - graphviz
+  - markupsafe
+  - psutil
+  - conda-forge::ncurses
+  - conda-forge::suitesparse
+
+# external software tools for chemistry
+  - coolprop
   - cantera::cantera=2.6
+  - conda-forge::mopac
   - conda-forge::cclib >=1.6.3
-  - rmg::chemprop
-  - coolprop
+  - conda-forge::openbabel >= 3
+
+# general-purpose external software tools
+  - conda-forge::julia=1.8.5
```

Comment:
on second thoughts, that could be a separate pull request if you prefer (and don't mind the churn)

Reply:
> Dare we try `>=` and let the CI alert us if it breaks?

Move fast and break things, I say. Will try now.

> Can we put ` - conda-forge::pyjulia` here instead of the ` - rmg::pyjulia` down below? My impression from #2381 was that the answer is "yes" but maybe we can test it here?
> on second thoughts, that could be a separate pull request if you prefer (and don't mind the churn)

Every package we don't have to maintain is welcome - will try this as well.